cmd/shell.go

```diff
@@ -302,3 +299,24 @@ func DebugServer(addr string) {
 		log.Fatalf("unable to boot debug server: %v", err)
 	}
 }
+
+// ReadJSONFile takes a file path as an argument and attempts to
+// unmarshal the content of the file into a struct containing a
+// configuration of a boulder component.
+func ReadJSONFile(filename string, out interface{}) error {
+	configData, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(configData, out)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// VersionString produces a friendly Application version string. Duplicated
+// from AppShell, with the exception that it takes a name as an argument.
```

Comment:
small nit: can you reference the fully qualified package for `AppShell` in this comment? It's not a Boulder class should someone try to go looking for it.